bindings_ffi/src/mls.rs

```diff
@@ -550,10 +550,10 @@ impl FfiXmtpClient {
     }
 
     /// Manually trigger a device sync request to sync records from another active device on this account.
-    pub async fn send_sync_request(&self, kind: FfiDeviceSyncKind) -> Result<(), GenericError> {
+    pub async fn send_sync_request(&self) -> Result<(), GenericError> {
```

Comment:
Per my document early I think we need 4 different functions here
- send_history_transfer_request - (sends a request to get history)
- approve_history_transfer_request - (approves a request to give history and sends the history)
- fetch_history - (checks to see if history is present to receive and syncs if it is)
- sync_state - (syncs the sync group for any new consent or hmac records *does not do anything with history transfer requests if called)

Reply:
Sounds great, let's address those specifically in a follow-up PR! Should be quick and easy.